metrics/mixin/alerts/perf.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ODF-ceph-mds-high-cpu-warnings.rules',
```

Comment:
A new alert group is already defined by #2309 , please put this rule under that group.

Reply:
Ack